databaseFunctions.py

```diff
@@ -46,21 +57,34 @@ def addJsonToTable(cursor, conn, courseCode, columnName, data):
     conn.commit()
 
 
-def insertCourseIntoDatabase(name, code, prereqs, antireqs, coreqs, precoreqs, desc, location, extra, conn, cursor):
+def insertCategoryIntoDatabase(category, breadth, conn, cursor):
+    insertCatQuery = "INSERT INTO \"Category\" (category_name, breadth) VALUES (%s, %s)"
+    cursor.execute(insertCatQuery, (category, breadth))
```

Comment:
Shoot I didn't mention this in the fe pr but could we store the full category name here as well?

Reply:
oh good idea, yea ill do that